src/components/structures/dialog/dialog.component.tsx

```diff
@@ -31,6 +31,14 @@ export interface DialogProps {
    * @default false
    */
   disableEscapeKeyDown?: boolean;
+  /**
+   * If true, the dialog stretches to maxWidth.
+   */
+  fullWidth?: boolean;
+  /**
+   * Determine the max-width of the dialog. The dialog width grows with the size of the screen. Set to false to disable maxWidth.
+   */
+  maxWidth?: 'xs' | 'sm' | 'md' | 'lg' | 'xl' | false;
```

Comment:
Do we want to not allow string here? (MUI allows that but I'm just checking if not allowing string is intentional)